### ISSUE TITLE
Map rerenders after each zoom

### ISSUE THREAD

**Issue by bangonkali** [meteorclub/cityforks#1]
Any advice on removing the flicker after every zoom or when there's new data on the current view port?

**Comment by queso** [comment]
Yeah, kind of a big hammer to make it work at the last minute. My guess is the templates could create reactive vars for maps and markers and could handle rendering off that...

**Comment by bangonkali** [comment]
Did some work [here](https://github.com/bangonkali/meteor-mapping). You can now mark this as resolved.

**Comment by queso** [comment]
Thanks for the link, but I don't really see how you resolved this issue. My plan is to work on this still, so I am going to keep this open :)

**Comment by bangonkali** [comment]
Yep. Really sorry. I think I shouldn't have closed it. My bad. 

Anyway, I think the critical part for a possible solution for leaflet maybe similar to an implementation I made [here](https://github.com/bangonkali/meteor-mapping/blob/master/common/globalRoutes.js). however it only uses gmaps api. btw, this method is taken from another project i found [here](https://github.com/avrora/meteor-gMaps).
